lib/game.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+require_relative 'ui.rb'
+
+class Game
+  attr_accessor :continue
+  def initialize(board, player1, player2, ui)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+    @ui = ui
+    @ui.show_board(@new_board)
+    play
+  end
+
+  def play
+    @player1.assign_sign(@ui)
+    @player2.sign = @player1.sign == 'X' ? 'O' : 'X'
+    while @new_board.check_board
+      cell = @ui.choose_num(@new_board)
```

Comment:
Lines 22-25 is exactly the same as lines 33-36. It's a candidate that we should extract this outside of this method and have another method take care of this.